packages/config/config/devices/0x0005/pe653.json

```
@@ -15,45 +15,466 @@
+		},
+		{
+			"#": "3[0xff0000]",
+			"$import": "#paramInformation/3[0xff000000]",
+			"label": "Air/Freeze Temperature Offset"
+		},
+		{
+			"#": "3[0xff00]",
+			"$import": "#paramInformation/3[0xff000000]",
+			"label": "Solar Temperature Offset"
+		},
+		{
+			"#": "22",
+			"$import": "~/templates/master_template.json#base_options_nounit",
+			"label": "Pool/Spa Configuration",
+			"options": [
+				{
+					"label": "Pool",
+					"value": 0
+				},
+				{
+					"label": "Spa",
+					"value": 1
+				},
+				{
+					"label": "Both",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "23",
+			"$import": "~/templates/master_template.json#base_options_nounit",
+			"label": "Spa Mode Pump Speed",
+			"description": "Requires pool/spa configuration and connected variable speed pump.",
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 1
+				},
+				{
+					"label": "Circuit 1",
+					"value": 2
+				},
+				{
+					"label": "VSP Speed 1",
+					"value": 3
+				},
+				{
+					"label": "VSP Speed 2",
+					"value": 4
+				},
+				{
+					"label": "VSP Speed 3",
+					"value": 5
+				},
+				{
+					"label": "VSP Speed 4",
+					"value": 6
+				}
+			]
+		},
+		{
+			"#": "32",
+			"label": "Variable Speed Pump - Speed 1",
+			"description": "Requires connected variable speed pump.",
+			"valueSize": 2,
+			"unit": "RPM",
+			"minValue": 400,
+			"maxValue": 3450,
+			"defaultValue": 750,
+			"unsigned": true
+		},
+		{
+			"#": "33",
+			"$import": "#paramInformation/32",
+			"label": "Variable Speed Pump - Speed 2",
+			"defaultValue": 1500
+		},
+		{
+			"#": "34",
+			"$import": "#paramInformation/32",
+			"label": "Variable Speed Pump - Speed 3",
+			"defaultValue": 2350
+		},
+		{
+			"#": "35",
+			"$import": "#paramInformation/32",
+			"label": "Variable Speed Pump - Speed 4",
+			"defaultValue": 3110
+		},
+		{
+			"#": "49",
+			"$import": "#paramInformation/32",
+			"label": "Variable Speed Pump - Max Speed",
+			"defaultValue": 3450
+		},
+		{
+			"#": "50[0xff000000]",
+			"$import": "~/templates/master_template.json#base_options_nounit",
+			"label": "Freeze Protection: Temperature",
+			"description": "Temperature at air/freeze sensor when freeze protection starts. Set to 0 to disable freeze protection.",
+			"valueSize": 4,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "40 °F",
+					"value": 40
+				},
+				{
+					"label": "41 °F",
+					"value": 41
+				},
+				{
+					"label": "42 °F",
+					"value": 42
+				},
+				{
+					"label": "43 °F",
+					"value": 43
+				},
+				{
+					"label": "44 °F",
+					"value": 44
+				}
+			]
+		},
+		{
+			"#": "50[0x010000]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Freeze Protection: Circuit 1",
+			"description": "Turn on the circuit when system is in freeze protection mode. Requires connected air/freeze sensor.",
```

How about this?
```suggestion
			"label": "Freeze Protection: Turn On Circuit 1",
			"description": "Requires connected air/freeze sensor.",
```
with 5 identical parameters, the alternative would end up cluttering UIs a lot.